vaadin-context-menu-flow-parent/vaadin-context-menu-testbench/src/main/java/com/vaadin/flow/component/contextmenu/testbench/ContextMenuElement.java

```diff
@@ -29,4 +31,29 @@
 @Element("vaadin-context-menu")
 public class ContextMenuElement extends TestBenchElement {
 
+    /**
+     * This is an utility method, which will produce context click on the target
+     * element. If the target had ContextMenu, after opening the last
+     * ContextMenuOverlayElement can be used to find its menu items.
+     *
+     * @param target
+     *            The element to which the ContextMenu has been hooked to.
+     */
+    public static void openByRightClick(TestBenchElement target) {
+        Actions action = new Actions(target.getDriver());
+        action.contextClick(target).perform();
+    }
+
+    /**
+     * Check if the ContextMenu is open.
+     *
+     * @return boolean True if menu is open.
+     */
+    public boolean isOpen() {
+        try {
+            return getAttribute("opened").equals("true");
```

Comment:
Please consider using return getPropertyBoolean("opened"); here as in other overlay based elements (like `NotificationElement#isOpen`, `DialogElement#isOpen`)